src/librustdoc/html/format.rs

```diff
@@ -50,7 +50,7 @@ pub struct MutableSpace(pub clean::Mutability);
 #[derive(Copy, Clone)]
 pub struct RawMutableSpace(pub clean::Mutability);
 /// Wrapper struct for emitting a where clause from Generics.
-pub struct WhereClause<'a>(pub &'a clean::Generics);
+pub struct WhereClause<'a>(pub &'a clean::Generics, pub String);
```

Comment:
I'd be in favor of making this:

```rust
pub struct WhereClause<'a> {
    pub generics: &'a clean::Generics,
    pub pad: String,
};
```

So we're explicit about what the fields are instead of a collection of unnamed data structures.